bot.py

```diff
@@ -51,7 +51,8 @@ def stage(self):
 
     # Callbacks
     def _on_look_to(self):
-        self._rhythm.initialise_line(self.stage, self._tower.user_controlled(0), time.time() + 3)
+        treble = Bell.from_number(1)
```

Comment:
@Kneasle I added this fix (as a seperate commit) with this pull request to avoid merge conflicts

Reply:
It might be worth just creating an entirely new branch/PR with the current version of master, because I've been adding non-PEP8 code to the master branch since 2 days ago and that won't be touched by the original commit.  And there'll also be no merge conflicts.

Reply:
I've already rebased this onto master (minus your latest commit)

Reply:
Aha OK cool